### PR TITLE
Amos | MPOC 76 missing functionality and bug fix

### PR DESCRIPTION
**Pull request by amosh101**
Patient attributes types have underscore.

### CODE REVIEW

File: ui/app/registration/controllers/patientCommonController.js [hunk 1]
```diff
@@ -291,6 +291,7 @@ angular.module('bahmni.registration')
 
             $scope.nationalityAttribute = function () {
                 $scope.patient.attribute = $scope.nationalAttribute;
+                console.log($scope.nationalAttribute);
```
Comment (nthfloor) [inline ui/app/registration/controllers/patientCommonController.js:294]: Do we need to keep this console.log() here or was it for debugging?

File: ui/app/registration/controllers/editPatientController.js [hunk 2]
```diff
@@ -27,8 +25,112 @@ angular.module('bahmni.registration')
...
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                        else if ($scope.nationalityChoice == 'Estrangeiro' || $scope.nationalityChoice == 'Foreigner') {
+                            $scope.nationalityDocs = foreignAttributes;
+                            console.log($scope.nationalityDocs);
+                            $scope.existDocs = $scope.nationalityDocs;
+                        }
+                    }
+                };
+                var existingPatientDocs = function () {
```
Comment (nthfloor) [inline ui/app/registration/controllers/editPatientController.js:67]: @amosh101 have you looked into adding unit tests for these new functions?

**Review by nthfloor — COMMENTED** [review]
@amosh101 left some comments

File: ui/app/registration/controllers/patientCommonController.js (hunk 1 above)
Comment (amosh101) [reply ui/app/registration/controllers/patientCommonController.js:294]: missed this one when cleaning the code, will remove it

File: ui/app/registration/controllers/editPatientController.js (hunk 2 above)
Comment (amosh101) [reply ui/app/registration/controllers/editPatientController.js:67]: It is a great idea, btw when can we schedule the call for the conventions and tips discussion?

Comment (nthfloor) [reply ui/app/registration/controllers/editPatientController.js:67]: Yes lets schedule this for next week Monday. For now lets leave the unit tests, but I'll create some tech debt tickets to go back and implement missing tests

**Pull request merged**